components/discovery/seed.json

```diff
@@ -45,9 +45,12 @@
   "pk": 4,
   "fields": {
     "combiner": 8,
+    "max_clients": 5,
     "timeout": 180,
-    "rounds": 5,
-    "clients_required": 2,
+    "rounds": 1,
+    "clients_required": 1,
+    "clients_requested": 5,
+    "algorithm": "fedavg",
```

Comment:
Great if we can set this as you have introduced on a combiner level. Will the reducer be responsible for this instruction?

Reply:
Yes, this is what I am thinking. Either the Reducer or some other entity.